src/engraving/accessibility/accessibleitem.cpp

```diff
@@ -42,11 +42,6 @@ AccessibleItem::~AccessibleItem()
 {
     m_element = nullptr;
 
-    AccessibleRoot* root = accessibleRoot();
-    if (root && root->focusedElement() == this) {
-        root->setFocusedElement(nullptr);
-    }
-
     if (m_registred && accessibilityController()) {
         accessibilityController()->unreg(this);
```

Comment:
undefined behaviour - accessibilityController->unreg( ) calls a virtual function on `this`, which is partially destructed here

Reply:
It is not about all virtual functions, see the case with Score::sigmap()
When we try to call the child method in EngravingItem::barbeat, the parent method is called